Projects/Services/APIKit/Sources/StudyRoomsAPI.swift

```diff
@@ -39,13 +40,16 @@ extension StudyRoomsAPI: DmsAPI {
 
         case .fetchMyStudyRoomApplicationItems:
             return "/my"
+        case .fetchStudyroomTimeList:
```

Comment:
스타일 가이드에 맞춰 줄바꿈이 필요할거같아요
```suggestion
            return "/my"

        case .fetchStudyroomTimeList:
```

Reply:
바꿨습니다